internal/scorecard/alpha/tests/bundle_test.go

```diff
@@ -117,3 +118,26 @@ func TestBasicAndOLM(t *testing.T) {
 
 	}
 }
+
+func TestOwnedCRDsForResources(t *testing.T) {
+	ownedCRDDescription := operators.CRDDescription{
+		Name:              "Test",
+		Version:           "v1",
+		Kind:              "Test",
+		StatusDescriptors: make([]operators.StatusDescriptor, 0),
+		Resources:         make([]operators.APIResourceReference, 0),
+	}
+
+	crd := operators.CustomResourceDefinitions{
+		Owned: []operators.CRDDescription{ownedCRDDescription},
+	}
+
+	res := scapiv1alpha2.ScorecardTestResult{}
+
+	res = CheckResources(crd, res)
+
+	if res.State != scapiv1alpha2.FailState {
+		t.Errorf("result State %v expected as owned CRDs do not have resources",
```

Comment:
```suggestion
		t.Errorf("Result State %v expected as owned CRDs do not have resources",
```

See that it is not passing in the CI: https://travis-ci.org/github/operator-framework/operator-sdk/jobs/681610538#L929


> Log messages do not begin with upper case:
> /home/travis/gopath/src/github.com/operator-framework/operator-sdk/internal/scorecard/alpha/tests/bundle_test.go:140:		t.Errorf("result State %v expected as owned CRDs do not have resources",

Reply:
solved